repository/d.json

```diff
@@ -346,6 +346,20 @@
 				}
 			]
 		},
+		{
+			"name": "Debian",
```

Comment:
The name is fine basically, at least it's not taken yet. But since "Debian" is a Linux distro, and not a specific language, just the word "Debian" doesn't say what the package does. Maybe "Debian package syntax" or "Debian Syntax"?

Reply:
Updated to `Debian Syntax`.